index.ftd

```diff
@@ -394,14 +395,14 @@ bg-color: $color.main.custom.ten
 
 
 
--- ftd.row colors:
+-- ftd.column colors:
 caption color-name:
 ftd.color bg-color:
 width: fill
 
 --- tp.h2: $color-name
 
---- ftd.row:
+--- ftd.column:
 width: 600
```

Comment:
@arcoprova we do not accept static widths unless it's advised specifically.

Reply:
Sir, I did not give the static width. It was already there before I have started working on it. Should I change it to fill?